## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 458692`
Commit: [`463f7a44c84241fadd6d0adeb999cfccc138d363`](https://github.com/NixOS/nixpkgs/commit/463f7a44c84241fadd6d0adeb999cfccc138d363) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/463f7a44c84241fadd6d0adeb999cfccc138d363..pull/458692/head))
Merge: [`69811444e0d6ddff622a73c08b80eb52d52e9dac`](https://github.com/NixOS/nixpkgs/commit/69811444e0d6ddff622a73c08b80eb52d52e9dac)

Logs: https://github.com/yzx9/nixpkgs-review-gha/actions/runs/19089541514


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 3 packages built:</summary>
  <ul>
    <li>servo</li>
    <li>taplo</li>
    <li>toml-test</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 3 packages built:</summary>
  <ul>
    <li>servo</li>
    <li>taplo</li>
    <li>toml-test</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>servo</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>taplo</li>
    <li>toml-test</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>servo</summary>
<pre>3ab91057ce9239.rlib,libasync_tungstenite-77edd22434bbd576.rlib,libtungstenite-f1e77f82bc61effb.rlib,libdata_encoding-d1b4b8cf46ef5fb0.rlib,libchrono-d7f89e621d936b5a.rlib,libiana_time_zone-0c048347bbfef76b.rlib,libimsz-e10058ece8edbcc1.rlib,libfst-81600332224c3c62.rlib,libasync_compression-210fe80c06423551.rlib,libcompression_codecs-12cc56c73f0b6ff2.rlib,libzstd-b52fc84b674c6403.rlib,libzstd_safe-3627f80a67b51e4e.rlib,libzstd_sys-d5cdf59f45ff6389.rlib,libcompression_core-6c1b1da6385529a1.rlib,libbrotli-b50e074a8bbd563f.rlib,libhyper_rustls-fbc06d0ee5a87266.rlib,libwebpki_roots-fea1a4826255e28d.rlib,libtokio_rustls-d922b53a6b36811a.rlib,librustls-186791becae1f9fb.rlib,libsubtle-6dfca59eb44a8a10.rlib,libwebpki-26a43146bf41ad46.rlib,libuntrusted-28ebc2d254daf90d.rlib,libaws_lc_rs-17b73e50c80fd379.rlib,libaws_lc_sys-6fd05005d9c9aaab.rlib,libhttp_body_util-0b07c4b36cf4a29f.rlib,libfutures-2d9432416d1b8826.rlib,libfutures_executor-dcdb0645fff99dae.rlib,liblayout_api-1b4c03a5a232acc6.rlib,libscript_traits-99c192fca2ef31f2.rlib,libhtml5ever-e19fedf44bd41e8f.rlib,libcanvas-d1b8af5e9fe3176c.rlib,libvello_cpu-6f32fb13bbb1326c.rlib,libvello_common-921dfaa8f7902f6d.rlib,libfearless_simd-00d7bab475899b3a.rlib,libfonts-5194de4af9ded787.rlib,libfontsan-c9437bc63be90ee9.rlib,liblibz_sys-69a58b3c10698fcf.rlib,libfontsan_woff2-4b88b09e92b5f8d6.rlib,libbrotli_decompressor-a5f0c54846ac4d40.rlib,liballoc_stdlib-e7f9ce1714b1f7e7.rlib,liballoc_no_stdlib-ba29476a299232bb.rlib,libharfbuzz_sys-7d46a0427e1364ab.rlib,libfreetype_sys-962d78b024bb7bd4.rlib,libskrifa-bfcee57b960e3d96.rlib,libpeniko-bab6f0537b6cf771.rlib,liblinebender_resource_handle-39d567c4771470ed.rlib,libcolor-194a5a37c500c3d8.rlib,libbluetooth_traits-43104247d2ba4bd9.rlib,libbackground_hang_monitor-7de2f31b58907e61.rlib,libbackground_hang_monitor_api-a9489849e713611e.rlib,libcompositing-141a97beb3f23060.rlib,libwebgpu-2ec1b3804888bbce.rlib,libtimers-7967c53ae1ff1c88.rlib,libwebgl-a59aba483023a2f6.rlib,libmedia-daa903eb2f72c104.rlib,libservo_media-d4dbb12c16aca547.rlib,libservo_media_webrtc-d7208b09c3ab0d98.rlib,libservo_media_audio-717798eb377c9063.rlib,libspeexdsp_resampler-f74fffa7490f0867.rlib,libpetgraph-4a3005969d5b5546.rlib,libordermap-2fdca59374d13209.rlib,libfixedbitset-1ab5627d20771ca8.rlib,libnum_complex-b8763117ba6f330e.rlib,libbyte_slice_cast-d03e4de8b5d5f362.rlib,libservo_media_player-745176b429947171.rlib,libservo_media_traits-f7f0d801aff6710d.rlib,libservo_media_streams-281ba81e2287b1d2.rlib,libwebxr-25a8af344804ce3c.rlib,libconstellation_traits-a86e1dc569ee3df3.rlib,libwebgpu_traits-03822acc7a4930cf.rlib,libwgpu_core-ce119311a61f9095.rlib,libwgpu_hal-4d12bd515de77e37.rlib,libnaga-4ddc1f2078eb043b.rlib,librustc_hash-73ccf4969783eb99.rlib,libhexf_parse-585e688a7f9e8a35.rlib,libbit_set-2610a42b9327c5e2.rlib,libbit_vec-74391be4df671b04.rlib,libcodespan_reporting-c9468b31f5f319bb.rlib,libunicode_width-df90795c3112aa61.rlib,libhalf-3c4a9ed9f3c53c24.rlib,libmetal-74e6043522d6fffc.rlib,libblock-9cc39b4f319670a0.rlib,libcore_graphics_types-66dcbb7a2c31a6ab.rlib,libcore_foundation-579e35fe5a1bccd9.rlib,libwgpu_types-04075055823e6639.rlib,libthiserror-103f27b2ea4f4c93.rlib,libstorage_traits-bc5a2155611ded02.rlib,libdevtools_traits-bf74a192c0a9450f.rlib,libnet_traits-329589219494ea79.rlib,librustls_pki_types-717fb36404384566.rlib,libzeroize-c80ff6f8cd6fc27c.rlib,libhyper_util-2f5ec5d185565a26.rlib,libcompositing_traits-0d6119c37b82f2da.rlib,libdpi-fd0e916ac6c8618a.rlib,libcanvas_traits-656912271b45d636.rlib,libwebxr_api-f01c263166b90db6.rlib,libembedder_traits-2defc794b4b8cd16.rlib,libwebdriver-9b56d13610fa4908.rlib,libtokio_stream-98f0bd2de0b5e515.rlib,libwarp-df8b4860683dbae5.rlib,libmime_guess-92306b296ce716f0.rlib,libunicase-47b9299ab9182c60.rlib,libserde_urlencoded-c0e463597f8888fb.rlib,libscoped_tls-70cc6c9ae5832cba.rlib,libheaders-f24c9daf86f4fb16.rlib,libheaders_core-9945bf44dc60345a.rlib,libbase64-210be848cb827ef8.rlib,libhyper-c9a4c0e6ac5b4545.rlib,libsocket2-2243804136ac33cf.rlib,libh2-6d2f81f8450bd480.rlib,libtower_service-5b76f013bf767f0b.rlib,libhttp_body-1f8f3991869865ed.rlib,libpin_project-2e8815eefb5784f4.rlib,libfutures_util-3e2fcd2761042826.rlib,libfutures_io-fbb894f4270127e8.rlib,libfutures_task-7667d96f33e39992.rlib,libhttp-97ac5b410dd9bba7.rlib,libcookie-db988e2c8270412a.rlib,libservo_geometry-dcbcaaa56fc724f7.rlib,libhyper_serde-74a543cfa41dfa25.rlib,libhyper-68e5e562ab4c4a16.rlib,libwant-7738a405bf550d4f.rlib,libtry_lock-44b98b1f5f6571d6.rlib,libhttparse-b2204bf883268cea.rlib,libh2-6102b12619bcac71.rlib,libtracing-e66b3e5484aff0b9.rlib,libtracing_core-be576c1ecffb13e9.rlib,libslab-a2bce2ddc17f6e0d.rlib,libtokio_util-13f47d1c81b2f282.rlib,libpin_utils-ea21a90c275b5d07.rlib,libatomic_waker-d032e63dbd712589.rlib,libfutures_channel-58d1456cec36de6f.rlib,libfutures_sink-48619a19f59485e1.rlib,libfutures_core-1baf7a3ba2393a6d.rlib,libhttp_body-ba78f63ef9c12086.rlib,libheaders-690976536821d47e.rlib,libhttpdate-088e6862fee9e4e6.rlib,libsha1-e2f3bf2742d594f8.rlib,libheaders_core-8f01d6c7b7ceba01.rlib,libcookie-ce7844a50cfe88dd.rlib,libstrum-ff5b968641aebc6d.rlib,libpixels-b9329406840470c9.rlib,libkurbo-46e529e38ec5b912.rlib,libfonts_traits-4bc9bfd641fdbf9b.rlib,librange-514e5c61cf873aa5.rlib,libprofile_traits-6a28d232f540a108.rlib,libread_fonts-9711f71605fba31a.rlib,libfont_types-d0bfb243f4df3703.rlib,libbase-64a26fcc01125d71.rlib,libservo_config-c35efe0acfebead7.rlib,libserde_json-e9d4571c53e1fdd5.rlib,libryu-215d731774995afa.rlib,libservo_url-b0c8fb08464c4547.rlib,libservo_malloc_size_of-54aee84ebe56c778.rlib,libwebrender-d73293e457da718a.rlib,libetagere-f8647097ed55acbc.rlib,libtopological_sort-abd30c04e843f5c9.rlib,libwr_glyph_rasterizer-7017108711907eee.rlib,libobjc-8e33dabc7d1cf7e3.rlib,libmalloc_buf-5a9af6244c9c3fa5.rlib,libcore_text-1c9f1c668870b3e0.rlib,libcore_graphics-6ff23bccb41560d9.rlib,libcore_graphics_types-9bd608b3e86bcef4.rlib,libforeign_types-9539db3278e5b561.rlib,libforeign_types_shared-a05fd5e6be00c7ea.rlib,libcore_foundation-84f60408b142c73c.rlib,libcore_foundation_sys-708808020e488f47.rlib,libwebrender_build-49d425b40bd5e2b5.rlib,libron-4de024257368e724.rlib,libunicode_ident-f9b155ec88b7522a.rlib,libplane_split-0be1d94ca6983097.rlib,libgleam-a08175c6d8ebd960.rlib,libsvg_fmt-735737da59ddc0b2.rlib,libtracy_rs-1772222f68d52085.rlib,libservo_allocator-cbc52f24f3cb7087.rlib,libtikv_jemallocator-1fa2778b1869ac2a.rlib,libtikv_jemalloc_sys-f3d853dfa1e78a28.rlib,libwebrender_api-77186340f41eda99.rlib,libzeitstempel-72ddf079e5373cae.rlib,libserde_bytes-db3813347a691a2f.rlib,libpeek_poke-5832f6acef42a7a9.rlib,libwr_malloc_size_of-8e37aa099b10a13e.rlib,liburlpattern-c95e4f026852a4f1.rlib,libunic_ucd_ident-4d8687182924c755.rlib,libunic_ucd_version-9536c0471e5d1d59.rlib,libunic_common-c6c3e0fc2f2b52d8.rlib,libunic_char_property-6f33622e8cf7ae70.rlib,libunic_char_range-60f7be6b93b5bca1.rlib,libtaffy-f96c808d15d1a5a1.rlib,libgrid-9b150442f9627a23.rlib,libresvg-a72d8f95e873a12c.rlib,libusvg-9b2f80b543947574.rlib,libunicode_vo-a9b9a10f7bca9bee.rlib,libimagesize-2bd155ac3b09aab4.rlib,libdata_url-37f569c26f749295.rlib,libxmlwriter-fb123d171dea8b23.rlib,librustybuzz-575c57d21e4077d1.rlib,libunicode_bidi_mirroring-b4ec75c4d6a1b0c9.rlib,libunicode_script-5c09117884d63730.rlib,libunicode_properties-5b95c5411fc28dff.rlib,libunicode_ccc-48b37228a8d9adaa.rlib,libsimplecss-5162984439ed910c.rlib,libroxmltree-ddd3c117745f7cdb.rlib,libsvgtypes-7bb883cfba07880e.rlib,libkurbo-d302b14d79fced6c.rlib,libfontdb-25d004216ad91529.rlib,libmemmap2-036c143a49722d07.rlib,libtinyvec-ce9a75b95a21562b.rlib,libtinyvec_macros-50ea6f6b4c986315.rlib,libslotmap-964f4ce02275cc90.rlib,libttf_parser-ab988f02684b0210.rlib,libtiny_skia-1d92a795815fc475.rlib,libtiny_skia_path-7c19695eb525a1b7.rlib,libstrict_num-fdb89caa3a2ba1bb.rlib,libfloat_cmp-8bb2c89493b837d0.rlib,libarrayref-ceb640cc5575aac2.rlib,libhttp-808e4116bfadc6f4.rlib,libcontent_security_policy-72974e43c214d708.rlib,libbase64-32975b4846b6b7b3.rlib,libonce_cell-fa03a468293bb7c9.rlib,libregex-5908637875fea13c.rlib,libregex_automata-51ea8c28be74762e.rlib,libaho_corasick-02dbc595478fd7c0.rlib,libregex_syntax-0bb25862ae2113e5.rlib,libsha2-b7ff5a723c114c8f.rlib,libcpufeatures-fa81cb3a4ab55011.rlib,libdigest-9ce998fef39246fe.rlib,libblock_buffer-747ff2073a282668.rlib,libcrypto_common-9571b748693a405c.rlib,libgeneric_array-29ec8748a8bacede.rlib,libtypenum-f77811f8e1eab32f.rlib,librand_core-9ce244494215ae7f.rlib,libgetrandom-6dbcade0bd7c0a5e.rlib,libaccountable_refcell-88d5fb4ea849a159.rlib,libtokio-822b5a2d383ffd5d.rlib,libsocket2-6ee34e1c050cc323.rlib,libbytes-887e22f48fb5bf5d.rlib,libmio-c7b451cb992deeb8.rlib,libpin_project_lite-549e2b1b34492c64.rlib,libmarkup5ever-3820af9690d23fdc.rlib,libtendril-ad10bad961d3c7f9.rlib,libutf8-b2d662752a678136.rlib,libfutf-db9053ed93ae8b29.rlib,libmac-945d25562474903f.rlib,libkeyboard_types-b3bc40221af5a140.rlib,libunicode_segmentation-2fef27dcd30df7ea.rlib,libuuid-da586d35dd349da4.rlib,libsha1_smol-cf1129dcaa5b228c.rlib,libstyle-f832ed981c7a4ed2.rlib,libencoding_rs-9c5db5a9d7a273e3.rlib,libstylo_config-ce59cdcba98eafc0.rlib,libnum_cpus-7a9e5e0a4a0604ed.rlib,libstylo_static_prefs-01f88ceda08269e2.rlib,libmime-afe106d89be7beb4.rlib,libicu_segmenter-cb8a0a28c256dd26.rlib,libicu_segmenter_data-1bfcf8d51c537a27.rlib,libcore_maths-c7f20f6479f3737f.rlib,liblibm-32de90b0e86fbb11.rlib,libbyteorder-679bcc9a8964d3ff.rlib,libuluru-49bca108b28e8586.rlib,libitertools-71311abd3bbec4f2.rlib,libstylo_dom-bf82f9f1978be400.rlib,libindexmap-0d21921c907ebd63.rlib,libhashbrown-de1cb82c6ce4d946.rlib,libfoldhash-89ed39bf5f445b4c.rlib,libequivalent-dffdf2f4552869ee.rlib,liballocator_api2-607b265cbaf1bd69.rlib,libstyle_traits-fc2b213aae3180d8.rlib,liburl-c5f34d7c145124be.rlib,libidna-ae1fd56d8b37b370.rlib,libidna_adapter-9f431b536bfaeb81.rlib,libicu_normalizer-db384ae51825ad29.rlib,libicu_normalizer_data-5fcb91797d7be04e.rlib,libwrite16-5f72d66f7c48edb2.rlib,libutf8_iter-457aa42232b3c0d2.rlib,libutf16_iter-85b48c55e3cdc9a4.rlib,libicu_properties-2423c4408c33c17e.rlib,libicu_properties_data-5fbd42430cc8606e.rlib,libicu_locid_transform-202a89a317a9d6ee.rlib,libicu_locid_transform_data-3c6bfd8c469d0ecb.rlib,libicu_collections-12ecabc4d347e63d.rlib,libicu_provider-3c839d78f65dec4d.rlib,libicu_locid-aa2d853776f9d85f.rlib,liblitemap-ab087999082ab467.rlib,libtinystr-9ee9955b6dd723e3.rlib,libzerovec-4003439a7f54432c.rlib,libwriteable-b4ab81db5d30a81f.rlib,libyoke-68e466486644c9f3.rlib,libzerofrom-d66ba8128d2036a1.rlib,libunicode_bidi-b823bb66ae41a853.rlib,libform_urlencoded-2a105e3226a04cfb.rlib,libpercent_encoding-208f4d06a11c6e0a.rlib,libatomic_refcell-b9270b53f667a99d.rlib,libstatic_assertions-d075e84ea8d29b05.rlib,libstylo_atoms-5a888978fb12fe27.rlib,libmatches-8aea6759d207d3d5.rlib,libweb_atoms-c1307de5f6cede27.rlib,libstylo_malloc_size_of-8de07d9cc4daf0ed.rlib,libvoid-d1fa37bf32e18b27.rlib,libselectors-027a7264934496c6.rlib,libto_shmem-977644e9d75934d6.rlib,libstring_cache-1e8d643e4e6af3ef.rlib,libthin_vec-fb7d101e03ec9d0a.rlib,libsmallbitvec-0e90fa1bb38ccafc.rlib,libprecomputed_hash-e5a400345ff32065.rlib,librustc_hash-79992536a0e8fe50.rlib,libservo_arc-7b85c9493ddb4546.rlib,libstable_deref_trait-e24fb829ae6899a2.rlib,libapp_units-34d1f2f52e7fd3e5.rlib,liblazy_static-e3e2a8eaabdcc74d.rlib,libderive_more-789d76064f1254ea.rlib,libcssparser-bb8ce95881ec80f4.rlib,libdtoa_short-447a1048269556f8.rlib,libdtoa-517d0a2290ee5416.rlib,libphf-2c5563bd4fde1488.rlib,libphf_shared-554c66a824fb4fb9.rlib,libsiphasher-d0e934da1e644036.rlib,libtime-ae191b1e4596f47f.rlib,libnum_threads-1cf704718a85da12.rlib,libitoa-ef344b548b8c1127.rlib,libtime_core-c1765ca49d380fa8.rlib,libnum_conv-e000de451b8ef425.rlib,libderanged-fd2a257df1cd767c.rlib,libpowerfmt-2665cfe4a2ccbdc3.rlib,libparking_lot-c592f0b6153d75cb.rlib,libparking_lot_core-adc88dfeb85b56d1.rlib,libsmallvec-70837d7caea78f13.rlib,liblock_api-b090629b01fcc6ef.rlib,libscopeguard-c44433d4c08500e0.rlib,libarboard-139353c080ceb51c.rlib,libimage-efad957a353447d0.rlib,libimage_webp-a1c6d8bbf086535b.rlib,libpng-65eb815eccbb55d4.rlib,libbitflags-53bf9d41e84e1cdc.rlib,libfdeflate-84fb2998cafb8b8c.rlib,libzune_jpeg-ce9059cde9a0abe8.rlib,libzune_core-42bb47199c768d59.rlib,libgif-c72018ae9c6bdab5.rlib,libcolor_quant-94e7662a9bd4d7dd.rlib,libbyteorder_lite-15f3697ea5ed78d3.rlib,libravif-49cb4ddf93765974.rlib,libloop9-4017931312af2293.rlib,libavif_serialize-446d4418a559f8e0.rlib,libquick_error-b488050cc39f459f.rlib,librgb-ca76d14e1d477374.rlib,librav1e-189b9b7d31723588.rlib,libprofiling-9bdd63531d769f85.rlib,libav1_grain-364e34b330c70e96.rlib,libnom-537ceea0ae3a15a8.rlib,libnum_rational-6045e47c1082810d.rlib,libnum_bigint-82b560e074edd40b.rlib,libnum_integer-3ce1dee5d28031b3.rlib,libanyhow-cea0e953cab4ad2e.rlib,libdebug_unreachable-842a29ec2aa01977.rlib,libmaybe_rayon-3ebe7836af3fdc2d.rlib,librayon-615a1b64cd5838e7.rlib,librayon_core-6500708e70e68a83.rlib,libcrossbeam_deque-04d29c5085e32380.rlib,libcrossbeam_epoch-eca243f9f1719f9e.rlib,libv_frame-01826238498bce41.rlib,libaligned_vec-00dde1fada565a14.rlib,libequator-8fc0deb4a8342190.rlib,libarrayvec-84a5951d6c5eb546.rlib,libthiserror-8fe02c7057f40413.rlib,libbitstream_io-f21eacf94abdce12.rlib,libitertools-fa2e8dbf063a3f3e.rlib,libeither-d6cecfec4575f511.rlib,libimgref-c48be7e7cf2181f9.rlib,libbytemuck-336c86e57b047010.rlib,libtiff-8168a0f7e5e48ffe.rlib,libflate2-c438cdeb7cf2a447.rlib,libminiz_oxide-f8d308f83d8c1190.rlib,libsimd_adler32-6eb8f7a77e148d0c.rlib,libcrc32fast-9db837b4a624fd7c.rlib,libweezl-e4d89fc93d81340e.rlib,libjpeg_decoder-d72c08a7eb76e104.rlib,libipc_channel-69ea7ff65ab5fffb.rlib,libcrossbeam_channel-03caed0c72d3f660.rlib,libcrossbeam_utils-7c2addb2ddea4e01.rlib,librand-1e2dcdf1a74616e6.rlib,librand_chacha-d3e7ac9a5849f4a6.rlib,libppv_lite86-67bfbf8e57eb0ced.rlib,libzerocopy-d013051004a56709.rlib,librand_core-e13549f4ff48909c.rlib,libgetrandom-1afa8b011bd30cae.rlib,libbincode-6c98ee711cd483b1.rlib,libsurfman-2ea910fd394dd34b.rlib,libfnv-b0d36070c7e3a7b8.rlib,libobjc2_core_video-2fa80ce65892eb0f.rlib,libmach2-a0ae4fe41da178ad.rlib,libobjc2_metal-34b9b0ffd4936d22.rlib,libobjc2_io_surface-518dfa12fa7de281.rlib,libcgl-ef1bfe5b1c24f432.rlib,libraw_window_handle-2a443ed33bc4cd6d.rlib,libobjc2_app_kit-87959fec20f987c8.rlib,libobjc2_quartz_core-a3f8f1366ebc907b.rlib,libobjc2_foundation-4393fbe7786b8c97.rlib,libobjc2_core_graphics-a223e001b2a4c4ed.rlib,libobjc2_core_foundation-74b8c975ead6089d.rlib,libobjc2-272712aafe268e87.rlib,libobjc2_encode-dc39dbef21852a4c.rlib,libglow-05e1a86c60720216.rlib,libeuclid-a442ecf73ed27b06.rlib,libserde-5545febf8fc24f32.rlib,libnum_traits-6cdd00bf2b846fbc.rlib,liblog-8eca87f9e8936b6b.rlib,libbitflags-04507b36d0bae7c5.rlib,libserde_core-afb10ecc368ad158.rlib,libsig-1be2206bb95a4204.rlib,libbacktrace-fe8fe326b7c4b052.rlib,libobject-899042ab6eea3ad4.rlib,libmemchr-e9690d5a558940bf.rlib,liblibc-46809137a3da9e31.rlib,libaddr2line-26d64e1b165e1444.rlib,libgimli-cba6a5622ff1bcc9.rlib,librustc_demangle-c6f1760e7eefe413.rlib,libcfg_if-74840df27a677180.rlib}.rlib&quot; &quot;&lt;sysroot&gt;/lib/rustlib/x86_64-apple-darwin/lib/{libstd-*,libpanic_unwind-*,libobject-*,libmemchr-*,libaddr2line-*,libgimli-*,librustc_demangle-*,libstd_detect-*,libhashbrown-*,librustc_std_workspace_alloc-*,libminiz_oxide-*,libadler2-*,libunwind-*,libcfg_if-*,liblibc-*,librustc_std_workspace_core-*,liballoc-*,libcore-*,libcompiler_builtins-*}.rlib&quot; &quot;-framework&quot; &quot;IOKit&quot; &quot;-lobjc&quot; &quot;-framework&quot; &quot;IOKit&quot; &quot;-framework&quot; &quot;ApplicationServices&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-framework&quot; &quot;CoreVideo&quot; &quot;-framework&quot; &quot;Carbon&quot; &quot;-framework&quot; &quot;AppKit&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-lc&quot; &quot;-lSystem&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-lobjc&quot; &quot;-lc++&quot; &quot;-lc++&quot; &quot;-lc++&quot; &quot;-lz&quot; &quot;-lc++&quot; &quot;-lc++&quot; &quot;-lfreetype&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-framework&quot; &quot;QuartzCore&quot; &quot;-framework&quot; &quot;Metal&quot; &quot;-lSystem&quot; &quot;-lSystem&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-lobjc&quot; &quot;-framework&quot; &quot;CoreText&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-framework&quot; &quot;CoreFoundation&quot; &quot;-framework&quot; &quot;OpenGL&quot; &quot;-framework&quot; &quot;CoreVideo&quot; &quot;-framework&quot; &quot;Metal&quot; &quot;-framework&quot; &quot;IOSurface&quot; &quot;-framework&quot; &quot;OpenGL&quot; &quot;-framework&quot; &quot;AppKit&quot; &quot;-framework&quot; &quot;QuartzCore&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-framework&quot; &quot;CoreFoundation&quot; &quot;-lobjc&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-liconv&quot; &quot;-lSystem&quot; &quot;-lc&quot; &quot;-lm&quot; &quot;-arch&quot; &quot;x86_64&quot; &quot;-mmacosx-version-min=11.3.0&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/servoshell-80c33c8a9426d0bb/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/tikv-jemalloc-sys-369aee083f3f59dc/out/build/lib&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/fontsan-e87f5dbdf8cd2862/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/fontsan-e87f5dbdf8cd2862/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/fontsan-e87f5dbdf8cd2862/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/fontsan-woff2-826274d9a76a5d81/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/harfbuzz-sys-6c65a07fe439f557/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/zstd-sys-d829bda6e752973b/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/aws-lc-sys-ece330ddc1ea32e0/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/task_info-01fe2cb85392e0a8/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/mozangle-e659316734577694/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/mozangle-e659316734577694/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/mozangle-e659316734577694/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/mozangle-e659316734577694/out&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/mozjs_sys-ffc85c454d275300/out/build/js/src/build&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/mozjs_sys-ffc85c454d275300/out/build&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/mozjs_sys-ffc85c454d275300/out/build&quot; &quot;-L&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/build/libsqlite3-sys-e891142c89b3c7e1/out&quot; &quot;-L&quot; &quot;/nix/store/yjxlzincrhac0fvc5dy2j0nbg0lqhqhp-zlib-1.3.1/lib&quot; &quot;-L&quot; &quot;/nix/store/cqy40l9zlxkf9mgxgpyx3zqx60yxrv2i-freetype-2.13.3/lib&quot; &quot;-o&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/deps/servo-38b5b64b3639e5c3&quot; &quot;-Wl,-dead_strip&quot; &quot;-nodefaultlibs&quot; &quot;-Wl,-rpath,@executable_path/lib/&quot; &quot;/nix/build/nix-9838-877865170/source/target/x86_64-apple-darwin/release/deps/servo_audit_data.o&quot; &quot;-Wl,-u,_AUDITABLE_VERSION_INFO&quot;
  = note: some arguments are omitted. use `--verbose` to show all linker arguments
  = note: Undefined symbols for architecture x86_64:
            &quot;typeinfo for js::BaseProxyHandler&quot;, referenced from:
                typeinfo for ForwardingProxyHandler in libmozjs_sys-1409323dc16f9145.rlib(702045c0c9daef17-jsglue.o)
            &quot;typeinfo for js::Wrapper&quot;, referenced from:
                typeinfo for WrapperProxyHandler in libmozjs_sys-1409323dc16f9145.rlib(702045c0c9daef17-jsglue.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

warning: servoshell@0.0.1: Inherited flag &quot;-fno-omit-frame-pointer&quot; is not supported by the currently used CC
warning: servoshell@0.0.1: Warning: supplying the --target x86_64-apple-macosx != x86_64-apple-darwin argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead.
warning: servoshell@0.0.1: Could not generate git version information: &quot;fatal: not a git repository (or any parent up to mount point /)\nStopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n&quot;
error: could not compile `servoshell` (bin &quot;servo&quot;) due to 1 previous error</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>servo</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>taplo</li>
    <li>toml-test</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>servo</summary>
<pre>ib,libtask_info-57ce05323cfecc1d.rlib,libnet-3cadfb436172454d.rlib,librustls_pemfile-2644eb2616027656.rlib,libasync_tungstenite-09eb1ac42e51ecb9.rlib,libtungstenite-a7ee82fd7df04ba0.rlib,libdata_encoding-f5443f3791f69edb.rlib,libchrono-0c9e0c731c6b983c.rlib,libiana_time_zone-4c6bfe72133d6949.rlib,libimsz-58932a4bb877f163.rlib,libfst-909fe12ff287d9df.rlib,libasync_compression-d1f2ca3034106650.rlib,libcompression_codecs-1b8882e6b5cc96a6.rlib,libzstd-b3a4ef718b8e8150.rlib,libzstd_safe-5fc69807b507ca2d.rlib,libzstd_sys-ef637da513b7488d.rlib,libcompression_core-3a048c60bcb7d6b1.rlib,libbrotli-282da1cdab438193.rlib,libhyper_rustls-662492433bd8baeb.rlib,libwebpki_roots-18afba6a55eae277.rlib,libtokio_rustls-4849f8d33e65f971.rlib,librustls-d9dfc03a05ed10c3.rlib,libsubtle-310a623c646dd7ec.rlib,libwebpki-f9bf6a9e2223b38d.rlib,libuntrusted-7c48ca60c7d5ccaf.rlib,libaws_lc_rs-f0b417dfbbe6489b.rlib,libaws_lc_sys-6a0a3ddcc01853cc.rlib,libhttp_body_util-4343e5469522c610.rlib,libfutures-f5b6491818cc4d22.rlib,libfutures_executor-51d3ed903ee5888e.rlib,liblayout_api-134ea23536908a01.rlib,libscript_traits-bfe643589344aee6.rlib,libhtml5ever-c7384c82e0f568be.rlib,libcanvas-9b8b0338e7c214d4.rlib,libvello_cpu-c53f3f1de63c2356.rlib,libvello_common-90da361a3a490d79.rlib,libfearless_simd-f4aa1433d7f9675e.rlib,libfonts-468ebf4aab8a3bad.rlib,libfontsan-ac7fc313fa7e931a.rlib,liblibz_sys-d6512c49b2864a43.rlib,libfontsan_woff2-eddb8e0e3f964968.rlib,libbrotli_decompressor-79dc9b2ff54d3f44.rlib,liballoc_stdlib-1503c64e6b5c259f.rlib,liballoc_no_stdlib-155a63a106d748a1.rlib,libharfbuzz_sys-6ca5fdf731c350eb.rlib,libfreetype_sys-6fea38f2706e2a5a.rlib,libskrifa-678e0d68cd96dbec.rlib,libpeniko-88d846376f576a5d.rlib,liblinebender_resource_handle-607c5858fb8dcc12.rlib,libcolor-6c7586c4965b451f.rlib,libbluetooth_traits-cba0c49abfbb287e.rlib,libbackground_hang_monitor-32807f83a148aae7.rlib,libbackground_hang_monitor_api-6241998751940186.rlib,libcompositing-ffc9942d1fd349ab.rlib,libwebgpu-f2cc9369c1ef3fb1.rlib,libtimers-995bf32b2642dac0.rlib,libwebgl-81c60689237abb99.rlib,libmedia-8c16bfca39672126.rlib,libservo_media-1864368303a662b9.rlib,libservo_media_webrtc-6e8a2b41049e864f.rlib,libservo_media_audio-ce946dec7cb79d17.rlib,libspeexdsp_resampler-6afaf81544f3e222.rlib,libpetgraph-b18995e2d39eeb55.rlib,libordermap-2682156a38ef35d1.rlib,libfixedbitset-ee7457a614df87e8.rlib,libnum_complex-f358a1f6ea79bdc7.rlib,libbyte_slice_cast-8ce2f4dca496bb18.rlib,libservo_media_player-b30fa4f15431792b.rlib,libservo_media_traits-b996eabda75ed8ba.rlib,libservo_media_streams-0661914f57dedab9.rlib,libwebxr-cbe3037945bba434.rlib,libconstellation_traits-6718a5830f021654.rlib,libwebgpu_traits-4b10e3abc50790b3.rlib,libwgpu_core-5f685366d0352eab.rlib,libwgpu_hal-53ebb68a9a13544a.rlib,libnaga-e89f6a6c036e84bd.rlib,librustc_hash-db0d7af823e9bea8.rlib,libhexf_parse-9053841f7c715ed2.rlib,libbit_set-f65b64529350ed53.rlib,libbit_vec-fec2d46878cbd579.rlib,libcodespan_reporting-1bbf098d1c604df0.rlib,libunicode_width-323976305c257992.rlib,libhalf-2f80d9f399451aef.rlib,libmetal-7b68954067557563.rlib,libblock-87719a85412007d1.rlib,libcore_graphics_types-dd033df44dd02eb8.rlib,libcore_foundation-fad92a2f82832592.rlib,libwgpu_types-e62aebc1b0426d63.rlib,libthiserror-8225e0bb4e9954b7.rlib,libstorage_traits-6ff88344715e87d8.rlib,libdevtools_traits-6e552d22bec1d253.rlib,libnet_traits-fcc3e909c73110f8.rlib,librustls_pki_types-09fa84ce6ab5636c.rlib,libzeroize-76ddf6d239c65752.rlib,libhyper_util-cceab7845d61ad95.rlib,libcompositing_traits-70be08dd9b096075.rlib,libdpi-6885cd1eae6549d2.rlib,libcanvas_traits-1a901f2217e4928d.rlib,libwebxr_api-63db027bdd630c5e.rlib,libembedder_traits-89c66c04df218ccf.rlib,libwebdriver-876aef9ca54507ad.rlib,libtokio_stream-492db3ee272d9e5c.rlib,libwarp-bc7abbb4fe09972d.rlib,libmime_guess-4cc2508c96fa0368.rlib,libunicase-8b77fdfd27af2018.rlib,libserde_urlencoded-83e1e4aefffcb75e.rlib,libscoped_tls-30e1906247a7b33a.rlib,libheaders-1c79fe2f9887ce02.rlib,libheaders_core-c714fcb6a4221089.rlib,libbase64-69c5596b7698b81c.rlib,libhyper-c3e2060945cb36b8.rlib,libsocket2-638e5a0c368d0451.rlib,libh2-3031be17f6b16b1e.rlib,libtower_service-a31bc6d8d5caaeba.rlib,libhttp_body-35f989151ca17c0c.rlib,libpin_project-9a42b112cb18910b.rlib,libfutures_util-e2809be57f67bef8.rlib,libfutures_io-553f5fe354227e53.rlib,libfutures_task-0a1237d704b76895.rlib,libhttp-d6b06a000354e646.rlib,libcookie-3cf37eab20007a59.rlib,libservo_geometry-f092b063720e666e.rlib,libhyper_serde-db3dfc47f712ca91.rlib,libhyper-b8271293f510badb.rlib,libwant-240ea2fa1a0fc76a.rlib,libtry_lock-2e09a613d0c4b8bf.rlib,libhttparse-1b633cdd1259d38b.rlib,libh2-8e1bdf4449e8ce40.rlib,libtracing-68afbca15c21a3ee.rlib,libtracing_core-14e262c259e143b4.rlib,libslab-f6ed2e1fea83e690.rlib,libtokio_util-5c8a4d9100692875.rlib,libpin_utils-4b97263b160987d8.rlib,libatomic_waker-7dccc0dec6d7ba8c.rlib,libfutures_channel-92d6409304a558d4.rlib,libfutures_sink-3df33a0dc56059a2.rlib,libfutures_core-8c1d1552f02dbc20.rlib,libhttp_body-56f273108d1744f0.rlib,libheaders-ac5cdd538ef8e06e.rlib,libhttpdate-f69fafe97e5b683e.rlib,libsha1-15bfa7816bd2a442.rlib,libheaders_core-2ca25f1cdc1502ac.rlib,libcookie-0e873c8a6c5b289e.rlib,libstrum-46e8a57d5202795a.rlib,libpixels-6418358f70b7592d.rlib,libkurbo-0945ad3b6da96429.rlib,libfonts_traits-370a1f17b6746684.rlib,librange-c0051e65f8e2cb4e.rlib,libprofile_traits-4abe8c92ec4ed698.rlib,libread_fonts-dc9c7834ea90f569.rlib,libfont_types-ea7885bcffcdcef4.rlib,libbase-0e30fe53a5778c00.rlib,libservo_config-fde424c80a1c75a1.rlib,libserde_json-64118b397beae407.rlib,libryu-37d51b08ad83eaa4.rlib,libservo_url-1b0d54bbe73eb2db.rlib,libservo_malloc_size_of-4a1adeab0fe041be.rlib,libwebrender-1f76f544be166e83.rlib,libetagere-712866001e7c3910.rlib,libtopological_sort-e376eeb60177ee56.rlib,libwr_glyph_rasterizer-65f14d46fd0ba8a4.rlib,libobjc-81125847be95fb68.rlib,libmalloc_buf-19cf2489ba97717d.rlib,libcore_text-b98367ffa865dc0c.rlib,libcore_graphics-864c921d7adbaca5.rlib,libcore_graphics_types-6d044812eac91686.rlib,libforeign_types-319dd2c96e0de38d.rlib,libforeign_types_shared-1eceb4aa1688878a.rlib,libcore_foundation-a554400257964cce.rlib,libcore_foundation_sys-6aee79fe3ec4cf9c.rlib,libwebrender_build-21a57e5d59fb3f40.rlib,libron-3a7e4a352e47114a.rlib,libunicode_ident-e6e1d2a30b8ef01e.rlib,libplane_split-c899a2a549cd75ee.rlib,libgleam-882aa6ef30e037f3.rlib,libsvg_fmt-5c4b01f99385fbde.rlib,libtracy_rs-cf7b6378430b99fa.rlib,libservo_allocator-7cc955ddc32bc1c9.rlib,libwebrender_api-e88bbe52130df7af.rlib,libzeitstempel-47e8f2cd50ba9a3a.rlib,libserde_bytes-ed3ee5a9514f4e90.rlib,libpeek_poke-5b32d08e111e7c8e.rlib,libwr_malloc_size_of-a7fca9b30c465cb6.rlib,liburlpattern-6689384ff8a21f7c.rlib,libunic_ucd_ident-b50876aad19ff925.rlib,libunic_ucd_version-9639dcc491eee554.rlib,libunic_common-c5bd54bb046f5e3f.rlib,libunic_char_property-2c5af52e66664b1c.rlib,libunic_char_range-d20cd12952c68619.rlib,libtaffy-768ab440c6d67625.rlib,libgrid-3d1fbbbe55bcf45e.rlib,libresvg-990d0c5724528707.rlib,libusvg-aa75d78c482d7f2f.rlib,libunicode_vo-017c69202190261f.rlib,libimagesize-ba6378ad0a2bac62.rlib,libdata_url-d9d768cb0a3f9987.rlib,libxmlwriter-d5dfae31d7536d8a.rlib,librustybuzz-b786e945ce63bcdc.rlib,libunicode_bidi_mirroring-88c532f417d54135.rlib,libunicode_script-4623541d8f278125.rlib,libunicode_properties-8eab62a5351a03f2.rlib,libunicode_ccc-0c873b05a8fd55b7.rlib,libsimplecss-b563df5ebce83765.rlib,libroxmltree-ff3d92b897ddc8a3.rlib,libsvgtypes-b50bc3c9ed386c84.rlib,libkurbo-710195ffa992fe2f.rlib,libfontdb-4995c7a108b3c91b.rlib,libmemmap2-65c58823419a092c.rlib,libtinyvec-edea58fdc94b8d5f.rlib,libtinyvec_macros-d9bc25f379542756.rlib,libslotmap-ce6509cedf4a9833.rlib,libttf_parser-a6e6a25f291752d3.rlib,libtiny_skia-863e3c0c1708f2ac.rlib,libtiny_skia_path-d6a5bd9aa586b5c3.rlib,libstrict_num-90f440282037023d.rlib,libfloat_cmp-0417dfa4d3928b91.rlib,libarrayref-7ba4debe0ee715d8.rlib,libhttp-7c71ad21011b0619.rlib,libcontent_security_policy-45fd76941f859d5a.rlib,libbase64-20d860891ba9b36d.rlib,libonce_cell-eb3f094370476161.rlib,libregex-8fd28e03de754121.rlib,libregex_automata-fd10a8fc77a773cd.rlib,libaho_corasick-9c61337a2ab53722.rlib,libregex_syntax-6291791803941617.rlib,libsha2-63f6a08c767a417e.rlib,libdigest-fbd8c95c55398eb6.rlib,libblock_buffer-8e0df16b7e4b1baf.rlib,libcrypto_common-c1ab4d969ebf1b34.rlib,libgeneric_array-99e857220569256e.rlib,libtypenum-67b668362dfae38a.rlib,librand_core-c804087ae89ccc50.rlib,libgetrandom-d62355222c7a59b1.rlib,libaccountable_refcell-999fc8f07030bb24.rlib,libtokio-316e2e2edbb4ed7f.rlib,libsocket2-3c4f017c344957f2.rlib,libbytes-6da085f334d36b75.rlib,libmio-9a55292489400c0a.rlib,libpin_project_lite-c0872f498474aa00.rlib,libmarkup5ever-82ed229642cefb3a.rlib,libtendril-b9dafbe46434c639.rlib,libutf8-821588ef9f3de44e.rlib,libfutf-9f13d4698f6e268a.rlib,libmac-a380c2c5bf862b29.rlib,libkeyboard_types-182fabad44306043.rlib,libunicode_segmentation-251cf6e228359b40.rlib,libuuid-c7882f532bdc68f3.rlib,libsha1_smol-57168e4bb6741112.rlib,libstyle-6ed42635596e80b6.rlib,libencoding_rs-492309ddb4cfb7e1.rlib,libstylo_config-654326fc1877dcf8.rlib,libnum_cpus-6a3fe720054ff756.rlib,libstylo_static_prefs-cbe2fc146feb0a8c.rlib,libmime-8ed44db85ef69e15.rlib,libicu_segmenter-50df9c733602a565.rlib,libicu_segmenter_data-2c86bd6d0d305be3.rlib,libcore_maths-9a13826bf1451d79.rlib,liblibm-abc1d634cf01e903.rlib,libbyteorder-3c69ea221e5819b1.rlib,libuluru-698916944ce393f6.rlib,libitertools-6e4390685c1b8796.rlib,libstylo_dom-b558055a1648c841.rlib,libindexmap-5d5ec620414606ef.rlib,libhashbrown-249bc5cb773e3b93.rlib,libfoldhash-e8b9589114eda685.rlib,libequivalent-be93b0dd9859d526.rlib,liballocator_api2-79eb1a7eeec2b94c.rlib,libstyle_traits-944b046ea9d9d037.rlib,liburl-bb32ba588e571957.rlib,libidna-0f472f4f695f4468.rlib,libidna_adapter-17f245a973d35d7c.rlib,libicu_normalizer-69335e9f8efcf4ce.rlib,libicu_normalizer_data-914446bca490cd48.rlib,libwrite16-0d6c7b107d627ec7.rlib,libutf8_iter-77d230aad470da17.rlib,libutf16_iter-e1dfbc08e1ee9881.rlib,libicu_properties-2ae43bf30c3c3428.rlib,libicu_properties_data-5c8fbaa7b1e998eb.rlib,libicu_locid_transform-cc4c379e856fce1c.rlib,libicu_locid_transform_data-9132661d51d75140.rlib,libicu_collections-b15df96158a30193.rlib,libicu_provider-118a6dc099da8ddc.rlib,libicu_locid-4a82d9cf83b9383e.rlib,liblitemap-d35326762d8f605f.rlib,libtinystr-927b16aedb7e2139.rlib,libzerovec-08566af32667617d.rlib,libwriteable-5bf094094aa22571.rlib,libyoke-1a4b4efa3ccbb6a7.rlib,libzerofrom-b23ac1896d4028a9.rlib,libunicode_bidi-f5552fcd3d858150.rlib,libform_urlencoded-6a491d1376016fdb.rlib,libpercent_encoding-c6759906aa875ec2.rlib,libatomic_refcell-9928d00578fae464.rlib,libstatic_assertions-1305cc1350880b3d.rlib,libstylo_atoms-395933bb8b738b09.rlib,libmatches-2b523b49ae66ab3d.rlib,libweb_atoms-c6aef2f0f5687e2b.rlib,libstylo_malloc_size_of-8b258268c912e911.rlib,libvoid-ab3fc16063d584f2.rlib,libselectors-ae418a2543bfe4ea.rlib,libto_shmem-491049b923102cd3.rlib,libstring_cache-5371e84e3089dfe4.rlib,libthin_vec-39ec46c283fb4404.rlib,libsmallbitvec-f6d847e71cd8e158.rlib,libprecomputed_hash-aa275f7f6a31c97a.rlib,librustc_hash-0d6bae2e5a750163.rlib,libservo_arc-9a3ee20ebb6a2c4a.rlib,libstable_deref_trait-0d60258295073944.rlib,libapp_units-77a0e0f7a5116321.rlib,liblazy_static-7be7ceabaf0c021f.rlib,libderive_more-051b386928a82f43.rlib,libcssparser-71f3039b9511fa06.rlib,libdtoa_short-a304131b071baeca.rlib,libdtoa-3cd8c9f22f97b358.rlib,libphf-4a1d1691006b56c3.rlib,libphf_shared-1b2e1cfe6bc6f8ce.rlib,libsiphasher-af65f49375d69b34.rlib,libtime-aa8c5b2abc7c3b32.rlib,libnum_threads-062d2ee01c523939.rlib,libitoa-b67d85b562cd3477.rlib,libtime_core-b13e67bb54949c65.rlib,libnum_conv-931cb1b8f3550cbb.rlib,libderanged-da57e14460b0a8d6.rlib,libpowerfmt-7c5ef9e80263d747.rlib,libparking_lot-478f9bbb4ac12be4.rlib,libparking_lot_core-53cea20a13c0d5be.rlib,libsmallvec-0e1c4fb09831a79e.rlib,liblock_api-e4c5be51eac0757d.rlib,libscopeguard-0fe4152d057c636c.rlib,libarboard-6e61975edbf917f8.rlib,libimage-aa6df62307fc1a5e.rlib,libimage_webp-844039fb954b191b.rlib,libpng-4ae1d0ec2ed6da42.rlib,libbitflags-88f8b50ce9983890.rlib,libfdeflate-0802f3b00fa56994.rlib,libzune_jpeg-5599226856d62a26.rlib,libzune_core-e850b9c83c23ece2.rlib,libgif-ffa19d748c1e1a1f.rlib,libcolor_quant-e368e09384ee97e7.rlib,libbyteorder_lite-c7dc1b05ac90f434.rlib,libravif-644258b6d92eb366.rlib,libloop9-77cc947e60a8e183.rlib,libavif_serialize-532ee57baf3e093e.rlib,libquick_error-96346f12c28d10fc.rlib,librgb-c7f91bb390f4f6ba.rlib,librav1e-d32e8d18bd19c99a.rlib,libprofiling-f5bf499d4a9b5b21.rlib,libav1_grain-76d2aa2bff56b639.rlib,libnom-7075a4682aeb1dc5.rlib,libnum_rational-25f7ba9fe2fbb80d.rlib,libnum_bigint-5422b9ff8b62721e.rlib,libnum_integer-a01e77137ad7e3e7.rlib,libanyhow-1394caecfa31cd7b.rlib,libdebug_unreachable-add34d68a52f5545.rlib,libmaybe_rayon-d3ff9042dd4d2d7c.rlib,librayon-f7b1fd2f844d9f8d.rlib,librayon_core-375f3c1a4dc603aa.rlib,libcrossbeam_deque-24d24917afd4d27d.rlib,libcrossbeam_epoch-a18f0154a825f541.rlib,libv_frame-a73f7dbf2368f2b6.rlib,libaligned_vec-9dc4e697cbb865ef.rlib,libequator-54a6108485276330.rlib,libarrayvec-d06769b57602d010.rlib,libthiserror-7eb37cebf8c77161.rlib,libbitstream_io-79baabf89011d139.rlib,libitertools-bfe9e8041e2c5c54.rlib,libeither-7051dfee4248511a.rlib,libimgref-97a14470e32ba20d.rlib,libbytemuck-66033d4fefa9fd00.rlib,libtiff-a9cf93de1bf4a663.rlib,libflate2-e5f26d136e5884b2.rlib,libminiz_oxide-ab27225e0564fa6d.rlib,libsimd_adler32-3cd67877c99dcd31.rlib,libcrc32fast-45ae5ed2ce596646.rlib,libweezl-f61ca3cfe5d9a4b6.rlib,libjpeg_decoder-993d6360383b0d13.rlib,libipc_channel-c194e8ba7a2a01fe.rlib,libcrossbeam_channel-6d95477ce4f33b77.rlib,libcrossbeam_utils-b5865528ecdd867c.rlib,librand-3f5750f85729be41.rlib,librand_chacha-4406fa3110ae40e8.rlib,libppv_lite86-9a95625bc2bbfcdc.rlib,libzerocopy-66109875328a6f04.rlib,librand_core-f858afa80866617f.rlib,libgetrandom-24b174d2f6d5c44c.rlib,libbincode-5ccaa0dbc0160491.rlib,libsurfman-4c1a25a5b4c832b1.rlib,libfnv-7c70c20ae57158e9.rlib,libobjc2_core_video-c2751180be6f0a63.rlib,libmach2-6d3c9bb538bef969.rlib,libobjc2_metal-964ec78a7fc329b9.rlib,libobjc2_io_surface-16ed1a97d6d7dfa4.rlib,libcgl-4502931bc1b4b82d.rlib,libraw_window_handle-11aa012b5318a0e6.rlib,libobjc2_app_kit-38e1889a15df1ce7.rlib,libobjc2_quartz_core-68c0fad180a7992c.rlib,libobjc2_foundation-7cbec3014b55c473.rlib,libobjc2_core_graphics-26179dfee9585816.rlib,libobjc2_core_foundation-04883092304ca686.rlib,libobjc2-337fe8f5a3021496.rlib,libobjc2_encode-e2112aeebb8d8687.rlib,libglow-cdac4c98cafc5516.rlib,libeuclid-86dd179a8defba8d.rlib,libserde-399f4b5d9bf0375f.rlib,libnum_traits-98b962251686a8dd.rlib,liblog-f849a5dd606e2b07.rlib,libbitflags-8ed8400964f227e6.rlib,libserde_core-0986aefaeac4f194.rlib,libsig-11e1d5dc0891cef1.rlib,libbacktrace-aa5c340b1e6e296b.rlib,libobject-385a29cf4e66ec96.rlib,libmemchr-e8d07599d7a24c94.rlib,liblibc-dfe7f82b1bcd89a9.rlib,libaddr2line-ba71d5c2f07cb704.rlib,libgimli-05a6cc36b9c3abc4.rlib,librustc_demangle-ada88791268111e0.rlib,libcfg_if-5c7fc5a2a6efa4f2.rlib}.rlib&quot; &quot;&lt;sysroot&gt;/lib/rustlib/aarch64-apple-darwin/lib/{libstd-*,libpanic_unwind-*,libobject-*,libmemchr-*,libaddr2line-*,libgimli-*,librustc_demangle-*,libstd_detect-*,libhashbrown-*,librustc_std_workspace_alloc-*,libminiz_oxide-*,libadler2-*,libunwind-*,libcfg_if-*,liblibc-*,librustc_std_workspace_core-*,liballoc-*,libcore-*,libcompiler_builtins-*}.rlib&quot; &quot;-framework&quot; &quot;IOKit&quot; &quot;-lobjc&quot; &quot;-framework&quot; &quot;IOKit&quot; &quot;-framework&quot; &quot;ApplicationServices&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-framework&quot; &quot;CoreVideo&quot; &quot;-framework&quot; &quot;Carbon&quot; &quot;-framework&quot; &quot;AppKit&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-lc&quot; &quot;-lSystem&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-lobjc&quot; &quot;-lc++&quot; &quot;-lc++&quot; &quot;-lc++&quot; &quot;-lz&quot; &quot;-lc++&quot; &quot;-lc++&quot; &quot;-lfreetype&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-framework&quot; &quot;QuartzCore&quot; &quot;-framework&quot; &quot;Metal&quot; &quot;-lSystem&quot; &quot;-lSystem&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-lobjc&quot; &quot;-framework&quot; &quot;CoreText&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-framework&quot; &quot;CoreFoundation&quot; &quot;-framework&quot; &quot;OpenGL&quot; &quot;-framework&quot; &quot;CoreVideo&quot; &quot;-framework&quot; &quot;Metal&quot; &quot;-framework&quot; &quot;IOSurface&quot; &quot;-framework&quot; &quot;OpenGL&quot; &quot;-framework&quot; &quot;AppKit&quot; &quot;-framework&quot; &quot;QuartzCore&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-framework&quot; &quot;CoreGraphics&quot; &quot;-framework&quot; &quot;CoreFoundation&quot; &quot;-lobjc&quot; &quot;-framework&quot; &quot;Foundation&quot; &quot;-liconv&quot; &quot;-lSystem&quot; &quot;-lc&quot; &quot;-lm&quot; &quot;-arch&quot; &quot;arm64&quot; &quot;-mmacosx-version-min=11.3.0&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/servoshell-43429122d053581a/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/tikv-jemalloc-sys-bd6533bf2baf06e9/out/build/lib&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/fontsan-45b79cef5210efac/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/fontsan-45b79cef5210efac/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/fontsan-45b79cef5210efac/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/fontsan-woff2-91db3c5e62899e64/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/harfbuzz-sys-6185eadfa035f013/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/zstd-sys-9db1e80f979176b2/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/aws-lc-sys-09d9f84b5192629e/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/task_info-fad32ebe578e9f87/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/mozangle-5a4fe5c72e25af82/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/mozangle-5a4fe5c72e25af82/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/mozangle-5a4fe5c72e25af82/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/mozangle-5a4fe5c72e25af82/out&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/mozjs_sys-809716bb2d5ceaa7/out/build/js/src/build&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/mozjs_sys-809716bb2d5ceaa7/out/build&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/mozjs_sys-809716bb2d5ceaa7/out/build&quot; &quot;-L&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/build/libsqlite3-sys-ec2d21393a1a169f/out&quot; &quot;-L&quot; &quot;/nix/store/jyq4rjvwky8v61zi1s8j3ibzhgizccfr-zlib-1.3.1/lib&quot; &quot;-L&quot; &quot;/nix/store/balxdpv5kagaq1lds5q9yjpvaa3rqjkh-freetype-2.13.3/lib&quot; &quot;-o&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/deps/servo-cb63e48096ae8117&quot; &quot;-Wl,-dead_strip&quot; &quot;-nodefaultlibs&quot; &quot;-Wl,-rpath,@executable_path/lib/&quot; &quot;/nix/build/nix-6441-1102413866/source/target/aarch64-apple-darwin/release/deps/servo_audit_data.o&quot; &quot;-Wl,-u,_AUDITABLE_VERSION_INFO&quot;
  = note: some arguments are omitted. use `--verbose` to show all linker arguments
  = note: Undefined symbols for architecture arm64:
            &quot;typeinfo for js::BaseProxyHandler&quot;, referenced from:
                typeinfo for ForwardingProxyHandler in libmozjs_sys-aaa8b4d8213d86fd.rlib(702045c0c9daef17-jsglue.o)
            &quot;typeinfo for js::Wrapper&quot;, referenced from:
                typeinfo for WrapperProxyHandler in libmozjs_sys-aaa8b4d8213d86fd.rlib(702045c0c9daef17-jsglue.o)
          ld: symbol(s) not found for architecture arm64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

warning: servoshell@0.0.1: Inherited flag &quot;-fno-omit-frame-pointer&quot; is not supported by the currently used CC
warning: servoshell@0.0.1: Warning: supplying the --target arm64-apple-macosx != arm64-apple-darwin argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead.
warning: servoshell@0.0.1: Could not generate git version information: &quot;fatal: not a git repository (or any parent up to mount point /)\nStopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n&quot;
error: could not compile `servoshell` (bin &quot;servo&quot;) due to 1 previous error</pre>
</details>
</details>

--- a/pkgs/by-name/ta/taplo/package.nix
+++ b/pkgs/by-name/ta/taplo/package.nix
@@ -5,10 +5,16 @@
   fetchFromGitHub,
   pkg-config,
   openssl,
-  withLsp ? true,
   installShellFiles,
   versionCheckHook,
+
+  # passthru dependencies
   nix-update-script,
+  runCommand,
+  toml-test,
+
+  # Optional feature
+  withLsp ? true,
 }:
 
 rustPlatform.buildRustPackage (finalAttrs: {
@@ -52,7 +58,38 @@ rustPlatform.buildRustPackage (finalAttrs: {
   versionCheckProgramArg = "--version";
   doInstallCheck = true;
 
-  passthru.updateScript = nix-update-script { };
+  passthru = {
+    updateScript = nix-update-script { };
+
+    tests = {
+      toml-test =
+        let
+          # Unfortunately, taplo does not yet pass all toml-test v1.6.0 tests.
+          # Some of the failures are reported issues, others may not be.
+          # https://github.com/tamasfe/taplo/issues/486
+          skips = [
+            "valid/comment/nonascii"
+            "valid/datetime/edge"
+            "valid/key/quoted-unicode"
+            "valid/string/quoted-unicode"
+            "invalid/control/multi-cr"
+            "invalid/control/rawmulti-cr"
+            "invalid/table/super-twice"
+          ];
+        in
+        runCommand "taplo-toml-test"
+          {
+            nativeBuildInputs = [
+              finalAttrs.finalPackage
+              toml-test
+            ];
+          }
+          ''
+            toml-test taplo ${lib.concatMapStringsSep " " (a: "-skip ${a}") skips} -- toml-test
+            touch "$out"
+          '';
+    };
+  };
 
   meta = {
     description = "TOML toolkit written in Rust";

--- a/pkgs/by-name/ta/taplo/package.nix
+++ b/pkgs/by-name/ta/taplo/package.nix
@@ -2,7 +2,7 @@
   stdenv,
   lib,
   rustPlatform,
-  fetchCrate,
+  fetchFromGitHub,
   pkg-config,
   openssl,
   withLsp ? true,
@@ -15,13 +15,22 @@ rustPlatform.buildRustPackage (finalAttrs: {
   pname = "taplo";
   version = "0.10.0";
 
-  src = fetchCrate {
-    inherit (finalAttrs) version;
-    pname = "taplo-cli";
-    hash = "sha256-iKc4Nu7AZE1LSuqXffi3XERbOqZMOkI3PV+6HaJzh4c=";
+  src = fetchFromGitHub {
+    owner = "tamasfe";
+    repo = "taplo";
+    tag = "release-taplo-cli-${finalAttrs.version}";
+    hash = "sha256-FW8OQ5TRUuQK8M2NDmp4c6p22jsHodxKqzOMrcdiqXU=";
   };
 
-  cargoHash = "sha256-tvijtB5fwOzQnnK/ClIvTbjCcMeqZpXcRdWWKZPIulM=";
+  cargoPatches = [
+    # Update reqwest to fix darwin sandboxing issues
+    # See also: https://github.com/tamasfe/taplo/pull/669
+    ./update-reqwest.patch
+  ];
+
+  cargoHash = "sha256-FMpGo+kRcNgDj4qwYvdQKGwGazUKKMIVq0HCYMrTql0=";
+
+  buildAndTestSubdir = "crates/taplo-cli";
 
   nativeBuildInputs = [
     installShellFiles
@@ -32,20 +41,12 @@ rustPlatform.buildRustPackage (finalAttrs: {
 
   buildFeatures = lib.optional withLsp "lsp";
 
-  postInstall =
-    lib.optionalString
-      (
-        stdenv.buildPlatform.canExecute stdenv.hostPlatform
-        &&
-          # Creation of the completions fails on Darwin platforms.
-          !stdenv.hostPlatform.isDarwin
-      )
-      ''
-        installShellCompletion --cmd taplo \
-          --bash <($out/bin/taplo completions bash) \
-          --fish <($out/bin/taplo completions fish) \
-          --zsh <($out/bin/taplo completions zsh)
-      '';
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    installShellCompletion --cmd taplo \
+      --bash <($out/bin/taplo completions bash) \
+      --fish <($out/bin/taplo completions fish) \
+      --zsh <($out/bin/taplo completions zsh)
+  '';
 
   nativeInstallCheckInputs = [ versionCheckHook ];
   versionCheckProgramArg = "--version";

--- a/pkgs/by-name/ta/taplo/update-reqwest.patch
+++ b/pkgs/by-name/ta/taplo/update-reqwest.patch
@@ -1,0 +1,1291 @@
+diff --git a/Cargo.lock b/Cargo.lock
+index ea88fad..01ecbb1 100644
+--- a/Cargo.lock
++++ b/Cargo.lock
+@@ -24,11 +24,11 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "e89da841a80418a9b391ebaea17f5c112ffaaa96f621d2c285b5174da76b9011"
+ dependencies = [
+  "cfg-if",
+- "getrandom",
++ "getrandom 0.2.12",
+  "once_cell",
+  "serde",
+  "version_check",
+- "zerocopy",
++ "zerocopy 0.7.32",
+ ]
+ 
+ [[package]]
+@@ -168,7 +168,7 @@ checksum = "5fd55a5ba1179988837d24ab4c7cc8ed6efdeff578ede0416b4225a5fca35bd0"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -179,7 +179,7 @@ checksum = "c980ee35e870bd1a4d2c8294d4c04d0499e67bca1e4b5cefcc693c2fa00caea9"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -218,6 +218,12 @@ version = "0.21.7"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "9d297deb1925b89f2ccc13d7635fa0714f12c87adce1c75356b39ca9b7178567"
+ 
++[[package]]
++name = "base64"
++version = "0.22.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "72b3254f16251a8381aa12e40e3c4d2f0199f8c6508fbecb9d91f575e0fbb8c6"
++
+ [[package]]
+ name = "beef"
+ version = "0.5.2"
+@@ -290,9 +296,9 @@ checksum = "a2ef034f05691a48569bd920a96c81b9d91bbad1ab5ac7c4616c1f6ef36cb79f"
+ 
+ [[package]]
+ name = "bytes"
+-version = "1.5.0"
++version = "1.10.1"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "a2bd12c1caf447e69cd4528f47f94d203fd2582878ecb9e9465484c4148a8223"
++checksum = "d71b6127be86fdcfddb610f7182ac57211d4b18a3e9c82eb2d17662f2227ad6a"
+ 
+ [[package]]
+ name = "cast"
+@@ -315,6 +321,12 @@ version = "1.0.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
+ 
++[[package]]
++name = "cfg_aliases"
++version = "0.2.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "613afe47fcd5fac7ccf1db93babcb082c5994d996f20b8b159f2ad1658eb5724"
++
+ [[package]]
+ name = "ciborium"
+ version = "0.2.2"
+@@ -382,7 +394,7 @@ dependencies = [
+  "heck",
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -591,15 +603,6 @@ version = "1.10.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "11157ac094ffbdde99aa67b23417ebdd801842852b500e395a45a9c0aac03e4a"
+ 
+-[[package]]
+-name = "encoding_rs"
+-version = "0.8.34"
+-source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "b45de904aa0b010bce2ab45264d0631681847fa7b6f2eaa7dab7619943bc4f59"
+-dependencies = [
+- "cfg-if",
+-]
+-
+ [[package]]
+ name = "equator"
+ version = "0.2.2"
+@@ -617,7 +620,7 @@ checksum = "3bf679796c0322556351f287a51b49e48f7c4986e727b5dd78c972d30e2e16cc"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -773,7 +776,7 @@ checksum = "87750cf4b7a4c0625b1529e4c543c2182106e4dedc60a2a6455e00d212c489ac"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -829,6 +832,20 @@ dependencies = [
+  "wasm-bindgen",
+ ]
+ 
++[[package]]
++name = "getrandom"
++version = "0.3.4"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "899def5c37c4fd7b2664648c28120ecec138e4d395b459e5ca34f9cce2dd77fd"
++dependencies = [
++ "cfg-if",
++ "js-sys",
++ "libc",
++ "r-efi",
++ "wasip2",
++ "wasm-bindgen",
++]
++
+ [[package]]
+ name = "gimli"
+ version = "0.28.1"
+@@ -854,25 +871,6 @@ dependencies = [
+  "regex-syntax 0.8.5",
+ ]
+ 
+-[[package]]
+-name = "h2"
+-version = "0.3.26"
+-source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "81fe527a889e1532da5c525686d96d4c2e74cdd345badf8dfef9f6b39dd5f5e8"
+-dependencies = [
+- "bytes",
+- "fnv",
+- "futures-core",
+- "futures-sink",
+- "futures-util",
+- "http",
+- "indexmap",
+- "slab",
+- "tokio",
+- "tokio-util",
+- "tracing",
+-]
+-
+ [[package]]
+ name = "half"
+ version = "2.4.1"
+@@ -913,9 +911,9 @@ checksum = "7f24254aa9a54b5c858eaee2f5bccdb46aaf0e486a595ed5fd8f86ba55232a70"
+ 
+ [[package]]
+ name = "http"
+-version = "0.2.12"
++version = "1.3.1"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "601cbb57e577e2f5ef5be8e7b83f0f63994f25aa94d673e54a92d5c516d101f1"
++checksum = "f4a85d31aea989eead29a3aaf9e1115a180df8282431156e533de47660892565"
+ dependencies = [
+  "bytes",
+  "fnv",
+@@ -924,76 +922,107 @@ dependencies = [
+ 
+ [[package]]
+ name = "http-body"
+-version = "0.4.6"
++version = "1.0.1"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "7ceab25649e9960c0311ea418d17bee82c0dcec1bd053b5f9a66e265a693bed2"
++checksum = "1efedce1fb8e6913f23e0c92de8e62cd5b772a67e7b3946df930a62566c93184"
+ dependencies = [
+  "bytes",
+  "http",
+- "pin-project-lite",
+ ]
+ 
+ [[package]]
+-name = "httparse"
+-version = "1.8.0"
++name = "http-body-util"
++version = "0.1.3"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "d897f394bad6a705d5f4104762e116a75639e470d80901eed05a860a95cb1904"
++checksum = "b021d93e26becf5dc7e1b75b1bed1fd93124b374ceb73f43d4d4eafec896a64a"
++dependencies = [
++ "bytes",
++ "futures-core",
++ "http",
++ "http-body",
++ "pin-project-lite",
++]
+ 
+ [[package]]
+-name = "httpdate"
+-version = "1.0.3"
++name = "httparse"
++version = "1.10.1"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "df3b46402a9d5adb4c86a0cf463f42e19994e3ee891101b1841f30a545cb49a9"
++checksum = "6dbf3de79e51f3d586ab4cb9d5c3e2c14aa28ed23d180cf89b4df0454a69cc87"
+ 
+ [[package]]
+ name = "hyper"
+-version = "0.14.30"
++version = "1.6.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "a152ddd61dfaec7273fe8419ab357f33aee0d914c5f4efbf0d96fa749eea5ec9"
++checksum = "cc2b571658e38e0c01b1fdca3bbbe93c00d3d71693ff2770043f8c29bc7d6f80"
+ dependencies = [
+  "bytes",
+  "futures-channel",
+- "futures-core",
+  "futures-util",
+- "h2",
+  "http",
+  "http-body",
+  "httparse",
+- "httpdate",
+  "itoa",
+  "pin-project-lite",
+- "socket2",
++ "smallvec",
+  "tokio",
+- "tower-service",
+- "tracing",
+  "want",
+ ]
+ 
+ [[package]]
+ name = "hyper-rustls"
+-version = "0.24.2"
++version = "0.27.7"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "ec3efd23720e2049821a693cbc7e65ea87c72f1c58ff2f9522ff332b1491e590"
++checksum = "e3c93eb611681b207e1fe55d5a71ecf91572ec8a6705cdb6857f7d8d5242cf58"
+ dependencies = [
+- "futures-util",
+  "http",
+  "hyper",
++ "hyper-util",
+  "rustls",
++ "rustls-pki-types",
+  "tokio",
+  "tokio-rustls",
++ "tower-service",
++ "webpki-roots",
+ ]
+ 
+ [[package]]
+ name = "hyper-tls"
+-version = "0.5.0"
++version = "0.6.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "d6183ddfa99b85da61a140bea0efc93fdf56ceaa041b37d553518030827f9905"
++checksum = "70206fc6890eaca9fde8a0bf71caa2ddfc9fe045ac9e5c70df101a7dbde866e0"
+ dependencies = [
+  "bytes",
++ "http-body-util",
+  "hyper",
++ "hyper-util",
+  "native-tls",
+  "tokio",
+  "tokio-native-tls",
++ "tower-service",
++]
++
++[[package]]
++name = "hyper-util"
++version = "0.1.17"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "3c6995591a8f1380fcb4ba966a252a4b29188d51d2b89e3a252f5305be65aea8"
++dependencies = [
++ "base64 0.22.1",
++ "bytes",
++ "futures-channel",
++ "futures-core",
++ "futures-util",
++ "http",
++ "http-body",
++ "hyper",
++ "ipnet",
++ "libc",
++ "percent-encoding",
++ "pin-project-lite",
++ "socket2 0.6.1",
++ "tokio",
++ "tower-service",
++ "tracing",
+ ]
+ 
+ [[package]]
+@@ -1042,6 +1071,16 @@ version = "2.9.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "8f518f335dce6725a761382244631d86cf0ccb2863413590b31338feb467f9c3"
+ 
++[[package]]
++name = "iri-string"
++version = "0.7.8"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "dbc5ebe9c3a1a7a5127f920a418f7585e9e758e911d0466ed004f393b0e380b2"
++dependencies = [
++ "memchr",
++ "serde",
++]
++
+ [[package]]
+ name = "is-terminal"
+ version = "0.4.12"
+@@ -1085,10 +1124,11 @@ checksum = "b1a46d1a171d865aa5f83f92695765caa047a9b4cbae2cbf37dbd613a793fd4c"
+ 
+ [[package]]
+ name = "js-sys"
+-version = "0.3.72"
++version = "0.3.82"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "6a88f1bda2bd75b0452a14784937d796722fdebfe50df998aeb3f0b7603019a9"
++checksum = "b011eec8cc36da2aab2d5cff675ec18454fad408585853910a202391cf9f8e65"
+ dependencies = [
++ "once_cell",
+  "wasm-bindgen",
+ ]
+ 
+@@ -1109,11 +1149,11 @@ checksum = "2a071f4f7efc9a9118dfb627a0a94ef247986e1ab8606a4c806ae2b3aa3b6978"
+ dependencies = [
+  "ahash",
+  "anyhow",
+- "base64",
++ "base64 0.21.7",
+  "bytecount",
+  "fancy-regex",
+  "fraction",
+- "getrandom",
++ "getrandom 0.2.12",
+  "iso8601",
+  "itoa",
+  "memchr",
+@@ -1137,9 +1177,9 @@ checksum = "e2abad23fbc42b3700f2f279844dc832adb2b2eb069b2df918f455c4e18cc646"
+ 
+ [[package]]
+ name = "libc"
+-version = "0.2.161"
++version = "0.2.177"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "8e9489c2807c139ffd9c1794f4af0ebe86a828db53ecdc7fea2111d0fed085d1"
++checksum = "2874a2af47a2325c2001a6e6fad9b16a53b802102b528163885171cf92b15976"
+ 
+ [[package]]
+ name = "linux-raw-sys"
+@@ -1195,6 +1235,12 @@ dependencies = [
+  "hashbrown",
+ ]
+ 
++[[package]]
++name = "lru-slab"
++version = "0.1.2"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "112b39cec0b298b6c1999fee3e31427f74f676e4cb9879ed1a121b43661a4154"
++
+ [[package]]
+ name = "lsp-async-stub"
+ version = "0.7.0"
+@@ -1202,7 +1248,7 @@ dependencies = [
+  "anyhow",
+  "async-trait",
+  "futures",
+- "getrandom",
++ "getrandom 0.2.12",
+  "lsp-types",
+  "rowan",
+  "serde",
+@@ -1257,12 +1303,6 @@ dependencies = [
+  "autocfg",
+ ]
+ 
+-[[package]]
+-name = "mime"
+-version = "0.3.17"
+-source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "6877bb514081ee2a7ff5ef9de3281f14a4dd4bceac4c09388074a6b5df8a139a"
+-
+ [[package]]
+ name = "minimal-lexical"
+ version = "0.2.1"
+@@ -1498,7 +1538,7 @@ checksum = "a948666b637a0f465e8564c73e89d4dde00d72d4d473cc972f390fc3dcee7d9c"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -1635,7 +1675,16 @@ dependencies = [
+  "smallvec",
+  "symbolic-demangle",
+  "tempfile",
+- "thiserror",
++ "thiserror 1.0.57",
++]
++
++[[package]]
++name = "ppv-lite86"
++version = "0.2.21"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "85eae3c4ed2f50dcfe72643da4befc30deadb458a9b590d720cde2f2b1e97da9"
++dependencies = [
++ "zerocopy 0.8.27",
+ ]
+ 
+ [[package]]
+@@ -1650,9 +1699,9 @@ dependencies = [
+ 
+ [[package]]
+ name = "proc-macro2"
+-version = "1.0.78"
++version = "1.0.103"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "e2422ad645d89c99f8f3e6b88a9fdeca7fabeac836b1002371c4367c8f984aae"
++checksum = "5ee95bc4ef87b8d5ba32e8b7714ccc834865276eab0aed5c9958d00ec45f49e8"
+ dependencies = [
+  "unicode-ident",
+ ]
+@@ -1666,6 +1715,61 @@ dependencies = [
+  "memchr",
+ ]
+ 
++[[package]]
++name = "quinn"
++version = "0.11.9"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "b9e20a958963c291dc322d98411f541009df2ced7b5a4f2bd52337638cfccf20"
++dependencies = [
++ "bytes",
++ "cfg_aliases",
++ "pin-project-lite",
++ "quinn-proto",
++ "quinn-udp",
++ "rustc-hash 2.1.1",
++ "rustls",
++ "socket2 0.5.5",
++ "thiserror 2.0.17",
++ "tokio",
++ "tracing",
++ "web-time",
++]
++
++[[package]]
++name = "quinn-proto"
++version = "0.11.13"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "f1906b49b0c3bc04b5fe5d86a77925ae6524a19b816ae38ce1e426255f1d8a31"
++dependencies = [
++ "bytes",
++ "getrandom 0.3.4",
++ "lru-slab",
++ "rand",
++ "ring",
++ "rustc-hash 2.1.1",
++ "rustls",
++ "rustls-pki-types",
++ "slab",
++ "thiserror 2.0.17",
++ "tinyvec",
++ "tracing",
++ "web-time",
++]
++
++[[package]]
++name = "quinn-udp"
++version = "0.5.14"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "addec6a0dcad8a8d96a771f815f0eaf55f9d1805756410b39f5fa81332574cbd"
++dependencies = [
++ "cfg_aliases",
++ "libc",
++ "once_cell",
++ "socket2 0.5.5",
++ "tracing",
++ "windows-sys 0.52.0",
++]
++
+ [[package]]
+ name = "quote"
+ version = "1.0.35"
+@@ -1675,6 +1779,41 @@ dependencies = [
+  "proc-macro2",
+ ]
+ 
++[[package]]
++name = "r-efi"
++version = "5.3.0"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "69cdb34c158ceb288df11e18b4bd39de994f6657d83847bdffdbd7f346754b0f"
++
++[[package]]
++name = "rand"
++version = "0.9.2"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "6db2770f06117d490610c7488547d543617b21bfa07796d7a12f6f1bd53850d1"
++dependencies = [
++ "rand_chacha",
++ "rand_core",
++]
++
++[[package]]
++name = "rand_chacha"
++version = "0.9.0"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "d3022b5f1df60f26e1ffddd6c66e8aa15de382ae63b3a0c1bfc0e4d3e3f325cb"
++dependencies = [
++ "ppv-lite86",
++ "rand_core",
++]
++
++[[package]]
++name = "rand_core"
++version = "0.9.3"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "99d9a13982dcf210057a8a78572b2217b667c3beacbf3a0d8b454f6f82837d38"
++dependencies = [
++ "getrandom 0.3.4",
++]
++
+ [[package]]
+ name = "rayon"
+ version = "1.8.1"
+@@ -1750,46 +1889,43 @@ checksum = "2b15c43186be67a4fd63bee50d0303afffcef381492ebe2c5d87f324e1b8815c"
+ 
+ [[package]]
+ name = "reqwest"
+-version = "0.11.27"
++version = "0.12.24"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "dd67538700a17451e7cba03ac727fb961abb7607553461627b97de0b89cf4a62"
++checksum = "9d0946410b9f7b082a427e4ef5c8ff541a88b357bc6c637c40db3a68ac70a36f"
+ dependencies = [
+- "base64",
++ "base64 0.22.1",
+  "bytes",
+- "encoding_rs",
+  "futures-core",
+- "futures-util",
+- "h2",
+  "http",
+  "http-body",
++ "http-body-util",
+  "hyper",
+  "hyper-rustls",
+  "hyper-tls",
+- "ipnet",
++ "hyper-util",
+  "js-sys",
+  "log",
+- "mime",
+  "native-tls",
+- "once_cell",
+  "percent-encoding",
+  "pin-project-lite",
++ "quinn",
+  "rustls",
+- "rustls-pemfile",
++ "rustls-pki-types",
+  "serde",
+  "serde_json",
+  "serde_urlencoded",
+  "sync_wrapper",
+- "system-configuration",
+  "tokio",
+  "tokio-native-tls",
+  "tokio-rustls",
++ "tower",
++ "tower-http",
+  "tower-service",
+  "url",
+  "wasm-bindgen",
+  "wasm-bindgen-futures",
+  "web-sys",
+  "webpki-roots",
+- "winreg",
+ ]
+ 
+ [[package]]
+@@ -1808,7 +1944,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "688c63d65483050968b2a8937f7995f443e27041a0f7700aa59b0822aedebb74"
+ dependencies = [
+  "cc",
+- "getrandom",
++ "getrandom 0.2.12",
+  "libc",
+  "spin",
+  "untrusted",
+@@ -1824,7 +1960,7 @@ dependencies = [
+  "countme",
+  "hashbrown",
+  "memoffset",
+- "rustc-hash",
++ "rustc-hash 1.1.0",
+  "text-size",
+ ]
+ 
+@@ -1840,6 +1976,12 @@ version = "1.1.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "08d43f7aa6b08d49f382cde6a7982047c3426db949b1424bc4b7ec9ae12c6ce2"
+ 
++[[package]]
++name = "rustc-hash"
++version = "2.1.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "357703d41365b4b27c590e3ed91eabb1b663f07c4c084095e60cbed4362dff0d"
++
+ [[package]]
+ name = "rustix"
+ version = "0.38.31"
+@@ -1855,32 +1997,36 @@ dependencies = [
+ 
+ [[package]]
+ name = "rustls"
+-version = "0.21.12"
++version = "0.23.34"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "3f56a14d1f48b391359b22f731fd4bd7e43c97f3c50eee276f3aa09c94784d3e"
++checksum = "6a9586e9ee2b4f8fab52a0048ca7334d7024eef48e2cb9407e3497bb7cab7fa7"
+ dependencies = [
+- "log",
++ "once_cell",
+  "ring",
++ "rustls-pki-types",
+  "rustls-webpki",
+- "sct",
++ "subtle",
++ "zeroize",
+ ]
+ 
+ [[package]]
+-name = "rustls-pemfile"
+-version = "1.0.4"
++name = "rustls-pki-types"
++version = "1.13.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "1c74cae0a4cf6ccbbf5f359f08efdf8ee7e1dc532573bf0db71968cb56b1448c"
++checksum = "94182ad936a0c91c324cd46c6511b9510ed16af436d7b5bab34beab0afd55f7a"
+ dependencies = [
+- "base64",
++ "web-time",
++ "zeroize",
+ ]
+ 
+ [[package]]
+ name = "rustls-webpki"
+-version = "0.101.7"
++version = "0.103.8"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "8b6275d1ee7a1cd780b64aca7726599a1dbc893b1e64144529e55c3c2f745765"
++checksum = "2ffdfa2f5286e2247234e03f680868ac2815974dc39e00ea15adc445d0aafe52"
+ dependencies = [
+  "ring",
++ "rustls-pki-types",
+  "untrusted",
+ ]
+ 
+@@ -1945,16 +2091,6 @@ version = "1.2.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "94143f37725109f92c262ed2cf5e59bce7498c01bcc1502d7b9afe439a4e9f49"
+ 
+-[[package]]
+-name = "sct"
+-version = "0.7.1"
+-source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "da046153aa2352493d6cb7da4b6e5c0c057d8a1d0a9aa8560baffdd945acd414"
+-dependencies = [
+- "ring",
+- "untrusted",
+-]
+-
+ [[package]]
+ name = "security-framework"
+ version = "2.9.2"
+@@ -2015,7 +2151,7 @@ checksum = "33c85360c95e7d137454dc81d9a4ed2b8efd8fbe19cee57357b32b9771fccb67"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -2049,7 +2185,7 @@ checksum = "0b2e6b945e9d3df726b65d6ee24060aff8e3533d431f677a9695db04eff9dfdb"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -2118,6 +2254,16 @@ dependencies = [
+  "windows-sys 0.48.0",
+ ]
+ 
++[[package]]
++name = "socket2"
++version = "0.6.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "17129e116933cf371d018bb80ae557e889637989d8638274fb25622827b03881"
++dependencies = [
++ "libc",
++ "windows-sys 0.60.2",
++]
++
+ [[package]]
+ name = "spin"
+ version = "0.9.8"
+@@ -2142,6 +2288,12 @@ version = "0.11.1"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "7da8b5736845d9f2fcb837ea5d9e2628564b3b043a70948a3f0b778838c5fb4f"
+ 
++[[package]]
++name = "subtle"
++version = "2.6.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "13c2bddecc57b384dee18652358fb23172facb8a2c51ccc10d74c157bdea3292"
++
+ [[package]]
+ name = "symbolic-common"
+ version = "12.10.0"
+@@ -2178,9 +2330,9 @@ dependencies = [
+ 
+ [[package]]
+ name = "syn"
+-version = "2.0.48"
++version = "2.0.108"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "0f3531638e407dfc0814761abb7c00a5b54992b849452a0646b7f65c9f770f3f"
++checksum = "da58917d35242480a05c2897064da0a80589a2a0476c9a3f2fdc83b53502e917"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+@@ -2189,29 +2341,11 @@ dependencies = [
+ 
+ [[package]]
+ name = "sync_wrapper"
+-version = "0.1.2"
+-source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "2047c6ded9c721764247e62cd3b03c09ffc529b2ba5b10ec482ae507a4a70160"
+-
+-[[package]]
+-name = "system-configuration"
+-version = "0.5.1"
+-source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "ba3a3adc5c275d719af8cb4272ea1c4a6d668a777f37e115f6d11ddbc1c8e0e7"
+-dependencies = [
+- "bitflags 1.3.2",
+- "core-foundation",
+- "system-configuration-sys",
+-]
+-
+-[[package]]
+-name = "system-configuration-sys"
+-version = "0.5.0"
++version = "1.0.2"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "a75fb188eb626b924683e3b95e3a48e63551fcfb51949de2f06a9d91dbee93c9"
++checksum = "0bf256ce5efdfa370213c1dabab5935a12e49f2c58d15e9eac2870d3b4f27263"
+ dependencies = [
+- "core-foundation-sys",
+- "libc",
++ "futures-core",
+ ]
+ 
+ [[package]]
+@@ -2239,7 +2373,7 @@ dependencies = [
+  "schemars",
+  "serde",
+  "serde_json",
+- "thiserror",
++ "thiserror 1.0.57",
+  "time",
+  "toml",
+  "tracing",
+@@ -2307,7 +2441,7 @@ dependencies = [
+  "sha1",
+  "tap",
+  "taplo",
+- "thiserror",
++ "thiserror 1.0.57",
+  "time",
+  "tokio",
+  "tracing",
+@@ -2352,7 +2486,7 @@ dependencies = [
+  "clap",
+  "console_error_panic_hook",
+  "futures",
+- "getrandom",
++ "getrandom 0.2.12",
+  "indexmap",
+  "js-sys",
+  "lsp-async-stub",
+@@ -2404,7 +2538,16 @@ version = "1.0.57"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "1e45bcbe8ed29775f228095caf2cd67af7a4ccf756ebff23a306bf3e8b47b24b"
+ dependencies = [
+- "thiserror-impl",
++ "thiserror-impl 1.0.57",
++]
++
++[[package]]
++name = "thiserror"
++version = "2.0.17"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "f63587ca0f12b72a0600bcba1d40081f830876000bb46dd2337a3051618f4fc8"
++dependencies = [
++ "thiserror-impl 2.0.17",
+ ]
+ 
+ [[package]]
+@@ -2415,7 +2558,18 @@ checksum = "a953cb265bef375dae3de6663da4d3804eee9682ea80d8e2542529b73c531c81"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
++]
++
++[[package]]
++name = "thiserror-impl"
++version = "2.0.17"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "3ff15c8ecd7de3849db632e14d18d2571fa09dfc5ed93479bc4485c7a517c913"
++dependencies = [
++ "proc-macro2",
++ "quote",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -2497,7 +2651,7 @@ dependencies = [
+  "num_cpus",
+  "parking_lot",
+  "pin-project-lite",
+- "socket2",
++ "socket2 0.5.5",
+  "tokio-macros",
+  "windows-sys 0.48.0",
+ ]
+@@ -2510,7 +2664,7 @@ checksum = "5b8a1e28f2deaa14e508979454cb3a223b10b938b45af148bc0986de36f1923b"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -2525,27 +2679,14 @@ dependencies = [
+ 
+ [[package]]
+ name = "tokio-rustls"
+-version = "0.24.1"
++version = "0.26.4"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "c28327cf380ac148141087fbfb9de9d7bd4e84ab5d2c28fbc911d753de8a7081"
++checksum = "1729aa945f29d91ba541258c8df89027d5792d85a8841fb65e8bf0f4ede4ef61"
+ dependencies = [
+  "rustls",
+  "tokio",
+ ]
+ 
+-[[package]]
+-name = "tokio-util"
+-version = "0.7.11"
+-source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "9cf6b47b3771c49ac75ad09a6162f53ad4b8088b76ac60e8ec1455b31a189fe1"
+-dependencies = [
+- "bytes",
+- "futures-core",
+- "futures-sink",
+- "pin-project-lite",
+- "tokio",
+-]
+-
+ [[package]]
+ name = "toml"
+ version = "0.7.8"
+@@ -2580,11 +2721,50 @@ dependencies = [
+  "winnow",
+ ]
+ 
++[[package]]
++name = "tower"
++version = "0.5.2"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "d039ad9159c98b70ecfd540b2573b97f7f52c3e8d9f8ad57a24b916a536975f9"
++dependencies = [
++ "futures-core",
++ "futures-util",
++ "pin-project-lite",
++ "sync_wrapper",
++ "tokio",
++ "tower-layer",
++ "tower-service",
++]
++
++[[package]]
++name = "tower-http"
++version = "0.6.6"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "adc82fd73de2a9722ac5da747f12383d2bfdb93591ee6c58486e0097890f05f2"
++dependencies = [
++ "bitflags 2.4.2",
++ "bytes",
++ "futures-util",
++ "http",
++ "http-body",
++ "iri-string",
++ "pin-project-lite",
++ "tower",
++ "tower-layer",
++ "tower-service",
++]
++
++[[package]]
++name = "tower-layer"
++version = "0.3.3"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "121c2a6cda46980bb0fcd1647ffaf6cd3fc79a013de288782836f6df9c48780e"
++
+ [[package]]
+ name = "tower-service"
+-version = "0.3.2"
++version = "0.3.3"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "b6bc1c9ce2b5135ac7f93c72918fc37feb872bdc6a5533a8b85eb4b86bfdae52"
++checksum = "8df9b6e13f2d32c91b9bd719c00d1958837bc7dec474d94952798cc8e69eeec3"
+ 
+ [[package]]
+ name = "tracing"
+@@ -2605,7 +2785,7 @@ checksum = "34704c8d6ebcbc939824180af020566b01a7c01f80641264eba0999f6c2b6be7"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
+ 
+ [[package]]
+@@ -2769,28 +2949,24 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "9c8d87e72b64a3b4db28d11ce29237c246188f4f51057d65a7eab63b7987e423"
+ 
+ [[package]]
+-name = "wasm-bindgen"
+-version = "0.2.100"
++name = "wasip2"
++version = "1.0.1+wasi-0.2.4"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "1edc8929d7499fc4e8f0be2262a241556cfc54a0bea223790e71446f2aab1ef5"
++checksum = "0562428422c63773dad2c345a1882263bbf4d65cf3f42e90921f787ef5ad58e7"
+ dependencies = [
+- "cfg-if",
+- "once_cell",
+- "rustversion",
+- "wasm-bindgen-macro",
++ "wit-bindgen",
+ ]
+ 
+ [[package]]
+-name = "wasm-bindgen-backend"
+-version = "0.2.100"
++name = "wasm-bindgen"
++version = "0.2.105"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "2f0a0651a5c2bc21487bde11ee802ccaf4c51935d0d3d42a6101f98161700bc6"
++checksum = "da95793dfc411fbbd93f5be7715b0578ec61fe87cb1a42b12eb625caa5c5ea60"
+ dependencies = [
+- "bumpalo",
+- "log",
+- "proc-macro2",
+- "quote",
+- "syn 2.0.48",
++ "cfg-if",
++ "once_cell",
++ "rustversion",
++ "wasm-bindgen-macro",
+  "wasm-bindgen-shared",
+ ]
+ 
+@@ -2808,9 +2984,9 @@ dependencies = [
+ 
+ [[package]]
+ name = "wasm-bindgen-macro"
+-version = "0.2.100"
++version = "0.2.105"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "7fe63fc6d09ed3792bd0897b314f53de8e16568c2b3f7982f468c0bf9bd0b407"
++checksum = "04264334509e04a7bf8690f2384ef5265f05143a4bff3889ab7a3269adab59c2"
+ dependencies = [
+  "quote",
+  "wasm-bindgen-macro-support",
+@@ -2818,22 +2994,22 @@ dependencies = [
+ 
+ [[package]]
+ name = "wasm-bindgen-macro-support"
+-version = "0.2.100"
++version = "0.2.105"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "8ae87ea40c9f689fc23f209965b6fb8a99ad69aeeb0231408be24920604395de"
++checksum = "420bc339d9f322e562942d52e115d57e950d12d88983a14c79b86859ee6c7ebc"
+ dependencies = [
++ "bumpalo",
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
+- "wasm-bindgen-backend",
++ "syn 2.0.108",
+  "wasm-bindgen-shared",
+ ]
+ 
+ [[package]]
+ name = "wasm-bindgen-shared"
+-version = "0.2.100"
++version = "0.2.105"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "1a05d73b933a847d6cccdda8f838a22ff101ad9bf93e33684f39c1f5f0eece3d"
++checksum = "76f218a38c84bcb33c25ec7059b07847d465ce0e0a76b995e134a45adcb6af76"
+ dependencies = [
+  "unicode-ident",
+ ]
+@@ -2848,11 +3024,24 @@ dependencies = [
+  "wasm-bindgen",
+ ]
+ 
++[[package]]
++name = "web-time"
++version = "1.1.0"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "5a6580f308b1fad9207618087a65c04e7a10bc77e02c8e84e9b00dd4b12fa0bb"
++dependencies = [
++ "js-sys",
++ "wasm-bindgen",
++]
++
+ [[package]]
+ name = "webpki-roots"
+-version = "0.25.4"
++version = "1.0.4"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "5f20c57d8d7db6d3b86154206ae5d8fba62dd39573114de97c2cb0578251f8e1"
++checksum = "b2878ef029c47c6e8cf779119f20fcf52bde7ad42a731b2a304bc221df17571e"
++dependencies = [
++ "rustls-pki-types",
++]
+ 
+ [[package]]
+ name = "winapi"
+@@ -2885,6 +3074,12 @@ version = "0.4.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
+ 
++[[package]]
++name = "windows-link"
++version = "0.2.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "f0805222e57f7521d6a62e36fa9163bc891acd422f971defe97d64e70d0a4fe5"
++
+ [[package]]
+ name = "windows-sys"
+ version = "0.48.0"
+@@ -2903,6 +3098,15 @@ dependencies = [
+  "windows-targets 0.52.0",
+ ]
+ 
++[[package]]
++name = "windows-sys"
++version = "0.60.2"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "f2f500e4d28234f72040990ec9d39e3a6b950f9f22d3dba18416c35882612bcb"
++dependencies = [
++ "windows-targets 0.53.5",
++]
++
+ [[package]]
+ name = "windows-targets"
+ version = "0.48.5"
+@@ -2933,6 +3137,23 @@ dependencies = [
+  "windows_x86_64_msvc 0.52.0",
+ ]
+ 
++[[package]]
++name = "windows-targets"
++version = "0.53.5"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "4945f9f551b88e0d65f3db0bc25c33b8acea4d9e41163edf90dcd0b19f9069f3"
++dependencies = [
++ "windows-link",
++ "windows_aarch64_gnullvm 0.53.1",
++ "windows_aarch64_msvc 0.53.1",
++ "windows_i686_gnu 0.53.1",
++ "windows_i686_gnullvm",
++ "windows_i686_msvc 0.53.1",
++ "windows_x86_64_gnu 0.53.1",
++ "windows_x86_64_gnullvm 0.53.1",
++ "windows_x86_64_msvc 0.53.1",
++]
++
+ [[package]]
+ name = "windows_aarch64_gnullvm"
+ version = "0.48.5"
+@@ -2945,6 +3166,12 @@ version = "0.52.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "cb7764e35d4db8a7921e09562a0304bf2f93e0a51bfccee0bd0bb0b666b015ea"
+ 
++[[package]]
++name = "windows_aarch64_gnullvm"
++version = "0.53.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "a9d8416fa8b42f5c947f8482c43e7d89e73a173cead56d044f6a56104a6d1b53"
++
+ [[package]]
+ name = "windows_aarch64_msvc"
+ version = "0.48.5"
+@@ -2957,6 +3184,12 @@ version = "0.52.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "bbaa0368d4f1d2aaefc55b6fcfee13f41544ddf36801e793edbbfd7d7df075ef"
+ 
++[[package]]
++name = "windows_aarch64_msvc"
++version = "0.53.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "b9d782e804c2f632e395708e99a94275910eb9100b2114651e04744e9b125006"
++
+ [[package]]
+ name = "windows_i686_gnu"
+ version = "0.48.5"
+@@ -2969,6 +3202,18 @@ version = "0.52.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "a28637cb1fa3560a16915793afb20081aba2c92ee8af57b4d5f28e4b3e7df313"
+ 
++[[package]]
++name = "windows_i686_gnu"
++version = "0.53.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "960e6da069d81e09becb0ca57a65220ddff016ff2d6af6a223cf372a506593a3"
++
++[[package]]
++name = "windows_i686_gnullvm"
++version = "0.53.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "fa7359d10048f68ab8b09fa71c3daccfb0e9b559aed648a8f95469c27057180c"
++
+ [[package]]
+ name = "windows_i686_msvc"
+ version = "0.48.5"
+@@ -2981,6 +3226,12 @@ version = "0.52.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "ffe5e8e31046ce6230cc7215707b816e339ff4d4d67c65dffa206fd0f7aa7b9a"
+ 
++[[package]]
++name = "windows_i686_msvc"
++version = "0.53.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "1e7ac75179f18232fe9c285163565a57ef8d3c89254a30685b57d83a38d326c2"
++
+ [[package]]
+ name = "windows_x86_64_gnu"
+ version = "0.48.5"
+@@ -2993,6 +3244,12 @@ version = "0.52.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "3d6fa32db2bc4a2f5abeacf2b69f7992cd09dca97498da74a151a3132c26befd"
+ 
++[[package]]
++name = "windows_x86_64_gnu"
++version = "0.53.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "9c3842cdd74a865a8066ab39c8a7a473c0778a3f29370b5fd6b4b9aa7df4a499"
++
+ [[package]]
+ name = "windows_x86_64_gnullvm"
+ version = "0.48.5"
+@@ -3005,6 +3262,12 @@ version = "0.52.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "1a657e1e9d3f514745a572a6846d3c7aa7dbe1658c056ed9c3344c4109a6949e"
+ 
++[[package]]
++name = "windows_x86_64_gnullvm"
++version = "0.53.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "0ffa179e2d07eee8ad8f57493436566c7cc30ac536a3379fdf008f47f6bb7ae1"
++
+ [[package]]
+ name = "windows_x86_64_msvc"
+ version = "0.48.5"
+@@ -3017,6 +3280,12 @@ version = "0.52.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "dff9641d1cd4be8d1a070daf9e3773c5f67e78b4d9d42263020c057706765c04"
+ 
++[[package]]
++name = "windows_x86_64_msvc"
++version = "0.53.1"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "d6bbff5f0aada427a1e5a6da5f1f98158182f26556f345ac9e04d36d0ebed650"
++
+ [[package]]
+ name = "winnow"
+ version = "0.5.40"
+@@ -3027,14 +3296,10 @@ dependencies = [
+ ]
+ 
+ [[package]]
+-name = "winreg"
+-version = "0.50.0"
++name = "wit-bindgen"
++version = "0.46.0"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "524e57b2c537c0f9b1e69f1965311ec12182b4122e45035b1508cd24d2adadb1"
+-dependencies = [
+- "cfg-if",
+- "windows-sys 0.48.0",
+-]
++checksum = "f17a85883d4e6d00e8a97c586de764dabcc06133f7f1d55dce5cdc070ad7fe59"
+ 
+ [[package]]
+ name = "zerocopy"
+@@ -3042,7 +3307,16 @@ version = "0.7.32"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "74d4d3961e53fa4c9a25a8637fc2bfaf2595b3d3ae34875568a5cf64787716be"
+ dependencies = [
+- "zerocopy-derive",
++ "zerocopy-derive 0.7.32",
++]
++
++[[package]]
++name = "zerocopy"
++version = "0.8.27"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "0894878a5fa3edfd6da3f88c4805f4c8558e2b996227a3d864f47fe11e38282c"
++dependencies = [
++ "zerocopy-derive 0.8.27",
+ ]
+ 
+ [[package]]
+@@ -3053,5 +3327,22 @@ checksum = "9ce1b18ccd8e73a9321186f97e46f9f04b778851177567b1975109d26a08d2a6"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+- "syn 2.0.48",
++ "syn 2.0.108",
+ ]
++
++[[package]]
++name = "zerocopy-derive"
++version = "0.8.27"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "88d2b8d9c68ad2b9e4340d7832716a4d21a22a1154777ad56ea55c51a9cf3831"
++dependencies = [
++ "proc-macro2",
++ "quote",
++ "syn 2.0.108",
++]
++
++[[package]]
++name = "zeroize"
++version = "1.8.2"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "b97154e67e32c85465826e8bcc1c59429aaaf107c1e4a9e53c8d8ccd5eff88d0"
+diff --git a/Cargo.toml b/Cargo.toml
+index d98197c..e1100e2 100644
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -30,7 +30,7 @@ lsp-types          = { version = "0.93.0" }
+ once_cell          = { version = "1.4" }
+ parking_lot        = { version = "0.12.0" }
+ regex              = { version = "1.5" }
+-reqwest            = { version = "0.11.9", default-features = false }
++reqwest            = { version = "0.12.24", default-features = false }
+ rowan              = { version = "0.15.3" }
+ schemars           = { version = "0.8.3" }
+ serde              = { version = "1" }

--- a/pkgs/by-name/to/toml-test/package.nix
+++ b/pkgs/by-name/to/toml-test/package.nix
@@ -1,0 +1,45 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "toml-test";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "toml-lang";
+    repo = "toml-test";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-jOFkSEDNvvx8svgyYYpAbveQsclMsQRKJ2ocA6ty1Kw=";
+  };
+
+  vendorHash = "sha256-yt5rwpYzO38wEUhcyG4G367Byek20Uz3u+buAazq/5A=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X zgo.at/zli.version=${finalAttrs.version}"
+  ];
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Language agnostic test suite for TOML parsers";
+    homepage = "https://github.com/toml-lang/toml-test";
+    changelog = "https://github.com/toml-lang/toml-test/releases/tag/v${finalAttrs.version}";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [
+      yzx9
+      defelo
+    ];
+    mainProgram = "toml-test";
+  };
+})

